core/contracts/common/FixedPoint.sol

```diff
@@ -97,6 +127,22 @@ library FixedPoint {
         return Unsigned(a.rawValue.mul(b));
     }
 
+    /** @dev Multiplies two `Unsigned`s, reverting on overflow, and ceil's the resultant product rather than the default floor behavior. */
+    function mulCeil(Unsigned memory a, Unsigned memory b) internal pure returns (Unsigned memory) {
+        uint mulRaw = a.rawValue.mul(b.rawValue);
+        uint mod = mulRaw.mod(FP_SCALING_FACTOR);
```

Comment:
I would write a comment about this, but instead of using `.mod()`, we can use `%` because we know `FP_SCALING_FACTOR` will never be `0`.

Reply:
done